tests/test_smoketest.py

```diff
@@ -122,6 +128,24 @@ def smoketest(self, starting_replica, checkout_bucket, test_bucket):
                 else:
                     self.fail("Timed out waiting for notification to arrive")
 
+        for replica in self.replicas:
+            with self.subTest(f'Testing Bundle Enumeration on {replica.name}'):
+                enumerate_bundles = list()
+                page_size = 10
+                first_page = self.get_bundle_enumerations(replica.name, page_size)
+                self.assertEquals(first_page['per_page'], 10)
```

Comment:
`assertEquals` is a deprecated alias
```suggestion
                self.assertEqual(first_page['per_page'], 10)
```

Reply:
Does this test pass on Python 3.x? It should fail as of 3.2, I thought...

Reply:
it passes, i changed it tho